ethereum_history_api/oracles/src/ethereum/transaction.ts

```diff
@@ -0,0 +1,104 @@
+import {
+  AccessList,
+  Hex,
+  Transaction,
+  TransactionBase,
+  TransactionEIP1559,
+  TransactionEIP2930,
+  TransactionEIP4844,
+  TransactionLegacy,
+  concatHex,
+  toRlp
+} from 'viem';
+import { RecursiveArray, txTypeToHex } from './receipt.js';
+import { encodeField } from '../noir/oracles/common/encode.js';
+
+type TxBase = Omit<TransactionBase, 'yParity'>;
+
+export class TxRlpEncoder {
+  public static txToFields(tx: Transaction): RecursiveArray<Hex> {
+    switch (tx.type) {
+      case 'legacy':
```

Comment:
It is inconsistent with `switch` by transaction type in `receipt.ts` there is additional `case 'eip155'` case matcher

Reply:
Remove eip155 transaction type as it's not used in ethereumjs-vm nor in viem